api/envoy/api/v2/cds.proto

```diff
@@ -640,6 +640,37 @@ message Cluster {
   // outgoing connections made by the cluster. Order matters as the filters are
   // processed sequentially as connection events happen.
   repeated cluster.Filter filters = 40;
+
+  // If present, tells the client where to send load reports via LRS.
+  //
+  // If multiple clusters are configured to use the same LRS server,
+  // then the data for all of those clusters may be sent on the same LRS
+  // stream.
+  //
+  // Note that if LRS is configured this way, then there is no need for
+  // the LRS server to send LoadStatsResponse messages on the LRS
+  // stream, because the data it would have sent is already configured
+  // here.
+  message LoadReportConfig {
+    // A management server endpoint to stream load stats to via
+    // *StreamLoadStats*. This must have :ref:`api_type
+    // <envoy_api_field_core.ApiConfigSource.api_type>` :ref:`GRPC
+    // <envoy_api_enum_value_core.ApiConfigSource.ApiType.GRPC>`.
+    core.ApiConfigSource lrs_server = 1;
```

Comment:
what clusters can be part of lrs_server.cluster_names ? would it have to match Cluster.name of the cluster whose member it is?

Reply:
I misread the doc, please ignore.